examples/abitest/module2/rust/src/lib.rs

```diff
@@ -37,15 +37,17 @@ pub fn main(in_handle: u64) -> i32 {
     oak::set_panic_hook();
 
     // We expect a single empty message holding a reply channel.
-    let in_channel = oak::ReadHandle { handle: in_handle };
+    let in_channel = oak::ReadHandle {
+        handle: oak::Handle::new(in_handle),
+    };
     let mut buf = Vec::<u8>::with_capacity(2);
     let mut handles = Vec::with_capacity(2);
     oak::channel_read(in_channel, &mut buf, &mut handles);
     if handles.len() != 1 {
         return oak::OakStatus::ERR_INTERNAL.value();
     }
     let out_handle = handles[0];
-    info!("backend node: in={}, out={}", in_handle, out_handle);
+    info!("backend node: in={}, out={:?}", in_handle, out_handle);
```

Comment:
Maybe implement `Display` for `oak::Handle` ?

Reply:
I would prefer `Debug`, since this should really not be needed for anything other than debugging (for similar reasons why we are trying to prevent code from naively serialising it too).

Reply:
I'm open to this, but as the type is an opaque wrapper there is some logical consistency to requiring a debugging-print to see inside

Reply:
Yeah, `Debug`-only does makes sense.